README.md

```diff
@@ -1,30 +1,129 @@
 # HardwareCheckout
+
 A simple website which will facilitate physical hardware checkout
 
-# Server
+## Server Installation
+
+Step 1 - git clone
+
+```
+git clone https://github.com/5hu5ky/HardwareCheckout.git
+```
+ 
+Step 2 - Install the server... 
+
+```
+cd HardwareCheckout
+chmod a+x ./install.sh
+./install.sh
+```
+
+
+## Operational Notes
+
+### General
+The service runs under its own user and virtual environment...
+
+The application is installed as a service and can started and stopped like this:
+
+```
+systemctl start HardwareCheckout
+systemctl stop HardwareCheckout
+```
+
+### Debugging
+
+If you want to debug the application:
+
+```
+journalctl -u HardwareCheckout
+```
+
+View the entries within the last five minutes:
+
+```
+journalctl -u HardwareCheckout --since=-5m
+```
+
+View the last 25 log entries:
+
+```
+journalctl -u HardwareCheckout -n 25
+```
+
+Tail the log:
+
+```
+journalctl -u HardwareCheckout -f
+```
+
+
+If you want to run it as a foreground service:
 
-Initial server set-up is done via install.sh - run that first... It does almost everything for you, but cook. Run it as root.
+```
+systemctl stop HardwareCheckout
+su chvapp
```

Comment:
chvapp won't have permissions to open a port on `80` or `443`. We need to think about this.

Reply:
start/stop is done by root or a sudo capable user...

Reply:
all systemctl functions are run by root or a sudo privileged account